css/types/angle.json

```diff
@@ -0,0 +1,262 @@
+{
+  "css": {
+    "types": {
+      "angle": {
+        "__compat": {
+          "description": "<code>&lt;angle&gt;</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/angle",
+          "support": {
+            "webview_android": {
+              "version_added": true
+            },
+            "chrome": {
+              "version_added": "2"
+            },
+            "chrome_android": {
+              "version_added": true
+            },
+            "edge": {
+              "version_added": true
+            },
+            "edge_mobile": {
+              "version_added": true
+            },
+            "firefox": {
+              "version_added": "3.6"
```

Comment:
Sure about 3.6? transform has been added in 3.5 and angles are used for some functions like skew()

Reply:
This is what is written in the current table. I'm no expert and I haven't invested time to do research.

Reply:
Ok, our goal is to migrate, not necessary to make the content perfect. We will have special activities for this in 2018.